torchaudio/transforms.py

```diff
@@ -365,3 +365,36 @@ def forward(self, waveform):
             return kaldi.resample_waveform(waveform, self.orig_freq, self.new_freq)
 
         raise ValueError('Invalid resampling method: %s' % (self.resampling_method))
+
+
+class ComputeDeltas(torch.jit.ScriptModule):
```

Comment:
if this depends on the jib mode, it's better to add a jit test to make sure that it works properly.

Reply:
Do you have an example of what you have in mind?

Reply:
You just need to add a text case to ensure every operator is traceable.

Reply:
There was already a test verifying that the shape of the output is correct. I added a test to verify that the output matches the functional.  Thanks for pointing out [this link](https://l.workplace.com/l.php?u=https%3A%2F%2Fgithub.com%2Fpytorch%2Fpytorch%2Fblob%2F00d967c39da15b1654c7ec3afa57f2a983c1e70c%2Ftest%2Ftest_jit.py%23L14033&h=AT3e-r0lqKPwcdpluC73rNGwvXCLZNbeSRbnRr8wMB77YlMdbZfrMnqqBLeArbmNDV7pDkYoKI7ADwRW4MSoLY12N94bE0brPfYc4CnCiUEPnlE41RKQlqh6kUzJOuYTL91wjyAPZme0XUmKRWeq5Q_6TECMx9EI3olwSg), but could you clarify what you would like to verify? If the module does not compile, the test would not run at all.